src/applications/edu-benefits/0994/components/PaymentReviewView.jsx

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
+import React from 'react';
+import { connect } from 'react-redux';
+
+class PaymentReviewView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      formData: props.formData,
+      name: props.name,
+    };
+  }
+
+  render() {
+    const { formData, name } = this.state;
+    let value = formData;
+    if (formData === undefined) {
+      const propName = `bank${name
+        .substring(0, 1)
+        .toUpperCase()}${name.substring(1)}`;
+      const prefillBankAccount = _.get(
```

Comment:
What happens if prefillBankAccount is empty?

Reply:
returns an empty object and the returns an empty string